docs/documentation/_data/deckhouse-ports.yml

```
@@ -13,22 +13,22 @@ groups:
     protocol: TCP
     description:
       en: Cluster API webhook handler
-      ru: Webhook-обработчик Cluster API
+      ru: Вебхук-обработчик Cluster API
   - ports: "4201"
     protocol: TCP
     description:
       en: VMware Cloud Director cloud provider webhook handler
-      ru: Webhook-обработчик для cloud-провайдера VMware Cloud Director
+      ru: Вебхук-обработчик для cloud-провайдера VMware Cloud Director
   - ports: "4223"
     protocol: TCP
     description:
       en: Deckhouse controller webhook handler
-      ru: Webhook-обработчик контроллера Deckhouse
+      ru: Вебхук-обработчик контроллера Deckhouse
 
 - group: MastersToNodes
   description:
     en: Master to nodes traffic
-    ru: Трафик между мастер-узлами и остальными узлами
+    ru: Трафик от мастер-узлов к узлам
```

```suggestion
    ru: Трафик от мастер-узлов к узлам
```
@Lada7878 Поправь согласно политике пожалуйста